base/interfaces/src/main/java/org/openscience/cdk/interfaces/IStereoElement.java

```diff
@@ -104,6 +104,9 @@
     /** ExtendedTetrahedral (e.g. 2,3-pentadiene) */
     public static final int AL   = 0x43_00;
 
+    /** ExtendedCisTrans (Cumulene) */
```

Comment:
Very minor thing, cumulene is an example, right? -> "e.g. Cumulene", similar to line 104?

Reply:
(ignore at will...)

Reply:
No... cumulene is the class of compounds.

https://en.wikipedia.org/wiki/Cumulene

Reply:
Reading that, there seems to be some controversy about the minimal number of double bonds, noting that heteroallenes are seen by some as (hetero)cumulenes too... :(